build.sbt

```diff
@@ -169,12 +169,12 @@ lazy val cats = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     name := "paiges-cats",
     moduleName := "paiges-cats",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-core" % "2.6.0",
-      "org.typelevel" %%% "cats-laws" % "2.6.0" % Test,
-      "org.typelevel" %%% "discipline-scalatest" % "2.1.3" % Test
+      "org.typelevel" %%% "cats-core" % "2.6.1",
+      "org.typelevel" %%% "cats-laws" % "2.6.1" % Test,
+      "org.typelevel" %%% "discipline-scalatest" % "2.1.5" % Test
     ),
     mimaPreviousArtifacts := {
-      if (isDotty.value) Set.empty
+      if (scalaVersion.value.startsWith("3")) Set.empty
```

Comment:
do we still need this? Or does mima still not support scala 3?

Reply:
Right now we still need it because there's no Scala 3 release yet. I'll cut that later this week. Then we can remove it.